stats.go

```diff
@@ -0,0 +1,1306 @@
+package webrtc
+
+import (
+	"time"
+)
+
+// A Stats object contains a set of statistics copies out of a monitored component
+// of the WebRTC stack at a specific time.
+type Stats interface {
```

Comment:
Does this need to be public?

Reply:
Not a big deal, but we are putting some implementation details in the users face by having these public.

I don't feel strongly, but would be nice if `godoc` doesn't show anything we did to make our code shorter

Reply:
I do want to keep the type around. Maybe i'll get rid of the methods:

```go
type Stats interface{}
```

I'm not sure Type() and Timestamp() are so useful anyway. Everyone will end up casting to the base type and they can find those values there.

The initial idea for this had to do with keeping the results of the not-yet-implemented `GetStats` method typesafe. It returns a map with polymorphic values `map[string]Stats`.